pymess/locale/cs/LC_MESSAGES/django.po

```diff
@@ -564,19 +564,23 @@ msgstr "vztažený objekt SMS zprávy"
 msgid "related objects of SMS messages"
 msgstr "vztažené objekty SMS zpráv"
 
-#: models/sms.py:68
+#: models/sms.py:62
+msgid "is secret"
+msgstr "tajný?"
```

Comment:
translation is the same as in core project

Reply:
please without question mark

Reply:
changed.